java/src/main/java/lesson03/exercise/Hello.java

```diff
@@ -67,8 +67,8 @@ public static void main(String[] args) {
             throw new IllegalArgumentException("Expecting one argument");
         }
         String helloTo = args[0];
-        Tracer tracer = Tracing.init("hello-world");
-        new Hello(tracer).sayHello(helloTo);
-        tracer.close();
+        try (Tracer tracer = Tracing.init("hello-world")) {
+            new Hello(tracer).sayHello(helloTo);
+        }
```

Comment:
I am not convinced it's a good pattern to promote in the tutorial. Most  applications using tracing will be implemented as long-running services, for which instantiating a tracer in this way is not applicable.

Reply:
@yurishkuro, I agree. I have just pushed a change that removes the direct reference of Jaeger's Tracer, and replaces it with io.opentracing.Tracer.